DataManagementSystem/Client/DataManager.py

```diff
@@ -904,9 +904,9 @@ def registerFile( self, fileTuple, catalog = '' ):
     :param tuple fileTuple: (lfn, physicalFile, fileSize, storageElementName, fileGuid, checksum )
     :param str catalog: catalog name
     """
-    if isinstance( fileTuple, ( list, dict, set, tuple ) ):
-      fileTuples = list( fileTuple )
-    else:
+    if isinstance( fileTuple, ( list, set ) ):
+      fileTuples = fileTuple
+    elif isinstance( fileTuple, tuple ):
```

Comment:
Thanks Andrei! I went too fast with Andre's suggestion but a tuple of tuples shard to detect ;-)